src/Command/FetchConferencesCommand.php

```diff
@@ -197,4 +197,18 @@ protected function updateExistingConference(Conference $existingConference, Conf
 
         return $updated;
     }
+
+    private function shouldBeIgnored(Conference $conference): bool
+    {
+        foreach ($this->conferenceFilterRepository->findAll() as $conferenceFilter) {
+            if (fnmatch($conferenceFilter->getName(), $conference->getName(), FNM_CASEFOLD)) {
+                $conference->setExcluded(true);
+
+                return true;
+            }
+        }
+        $conference->setExcluded(false);
```

Comment:
Lets move excluded = false into the conference constructor

Reply:
Actually excluded is false by default so I just removed this line